src/ser/impl/formatter/details/escape.zig

```diff
@@ -93,7 +93,8 @@ pub fn escapeChar(codepoint: u21, writer: anytype) !void {
 }
 
 /// Escapes characters of a UTF-8 encoded string using JSON escape sequences.
-pub fn escape(bytes: []const u8, writer: anytype, formatter: anytype) !void {
+pub fn escape(input: []const u8, writer: anytype, formatter: anytype) !void {
+    var bytes = std.mem.sliceTo(input, 0);
```

Comment:
Is it possible to make `bytes` `const` here? I don't believe it's modified anywhere in this function.